drivers/python/rethinkdb/docs.py

```diff
@@ -121,8 +121,8 @@
 	(rethinkdb.ast.DB.table_list, b"db.table_list() -> array\n\nList all table names in a database. The result is a list of strings.\n\n*Example* List all tables of the 'test' database.\n\n    r.db('test').table_list().run(conn)\n    \n"),
 	(rethinkdb.ast.RqlQuery.__add__, b'number + number -> number\nstring + string -> string\narray + array -> array\ntime + number -> time\n\nSum two numbers, concatenate two strings, or concatenate 2 arrays.\n\n*Example* It\'s as easy as 2 + 2 = 4.\n\n    > (r.expr(2) + 2).run(conn)\n    \n    4\n\n*Example* Strings can be concatenated too.\n\n    > (r.expr("foo") + "bar").run(conn)\n    \n    "foobar"\n\n*Example* Arrays can be concatenated too.\n\n    > (r.expr(["foo", "bar"]) + ["buzz"]).run(conn)\n    \n    [\'foo\', \'bar\', \'buzz\']\n\n*Example* Create a date one year from now.\n\n    r.now() + 365*24*60*60\n\n*Example* Use [args](http://rethinkdb.com/api/python/args) with `add` to sum multiple values.\n\n    > r.add(r.args([10, 20, 30])).run(conn)\n    \n    60\n\n*Example* Concatenate an array of strings with `args`.\n\n    > r.add(r.args([\'foo\', \'bar\', \'buzz\'])).run(conn)\n    \n    "foobarbuzz"\n'),
 	(rethinkdb.add, b'number + number -> number\nstring + string -> string\narray + array -> array\ntime + number -> time\n\nSum two numbers, concatenate two strings, or concatenate 2 arrays.\n\n*Example* It\'s as easy as 2 + 2 = 4.\n\n    > (r.expr(2) + 2).run(conn)\n    \n    4\n\n*Example* Strings can be concatenated too.\n\n    > (r.expr("foo") + "bar").run(conn)\n    \n    "foobar"\n\n*Example* Arrays can be concatenated too.\n\n    > (r.expr(["foo", "bar"]) + ["buzz"]).run(conn)\n    \n    [\'foo\', \'bar\', \'buzz\']\n\n*Example* Create a date one year from now.\n\n    r.now() + 365*24*60*60\n\n*Example* Use [args](http://rethinkdb.com/api/python/args) with `add` to sum multiple values.\n\n    > r.add(r.args([10, 20, 30])).run(conn)\n    \n    60\n\n*Example* Concatenate an array of strings with `args`.\n\n    > r.add(r.args([\'foo\', \'bar\', \'buzz\'])).run(conn)\n    \n    "foobarbuzz"\n'),
-	(rethinkdb.ast.RqlQuery.__and__, b'bool & bool -> bool\nr.and_(bool, bool) -> bool\nbool.and_(bool) -> bool\n\nCompute the logical "and" of two or more values. The `and_` command can be used as an infix operator after its first argument (`r.expr(True).and_(False)`) or given all of its arguments as parameters (`r.and_(True, False)`). The standard Python and operator, `&`, may also be used with ReQL.\n\n*Example* Return whether both `a` and `b` evaluate to true.\n\n    > a = True\n    > b = False\n    > (r.expr(a) & b).run(conn)\n    \n    False\n*Example* Return whether all of `x`, `y` and `z` evaluate to true.\n\n    > x = True\n    > y = True\n    > z = True\n    > r.and_(x, y, z).run(conn)\n    \n    True\n'),
```

Comment:
This is also auto-generated from the files in This file is auto generated. These changes should be made in the corresponding files in https://github.com/rethinkdb/docs/tree/master/api